lib/pf/Switch/Ubiquity/Unifi.pm

```diff
@@ -0,0 +1,170 @@
+package pf::Switch::Ubiquity::Unifi;
+
+=head1 NAME
+
+pf::Switch::Ubiquity::Unifi
+
+=head1 SYNOPSIS
+
+The pf::Switch::Ubiquity::Unifi module implements an object oriented interface to
+manage Unifi  controllers
+
+=head1 STATUS
+
+Developed and tested on Unifi controller version 5.4.14 with a UniFi AP-AC-Pro running 3.4.14.3413
+
+=head1 BUGS AND LIMITATIONS
+
+=cut
+
+use strict;
+use warnings;
+
+use base ('pf::Switch');
+
+use pf::violation qw(violation_count_reevaluate_access);
+use pf::constants::node qw($STATUS_UNREGISTERED);
+use pf::file_paths qw($var_dir);
+use pf::constants;
+use pf::util;
+use pf::node;
+
+# The port to reach the Unifi controller API
+our $UNIFI_API_PORT = "8443";
```

Comment:
I believe they are now named 'disconnectPort' and 'coaPort' instead of controllerPort which was deprecated

Could always use disconnectPort but that doesn't sound like the right parameter and didn't want to create another parameter